examples/seq2seq/finetune_trainer.py

```diff
@@ -220,11 +221,14 @@ def main():
         data_args.eval_beams = model.config.num_beams
 
     # set decoder_start_token_id for MBart
-    if model.config.decoder_start_token_id is None and isinstance(tokenizer, MBartTokenizer):
+    if model.config.decoder_start_token_id is None and isinstance(tokenizer, (MBartTokenizer, MBartTokenizerFast)):
         assert (
             data_args.tgt_lang is not None and data_args.src_lang is not None
         ), "mBart requires --tgt_lang and --src_lang"
-        model.config.decoder_start_token_id = tokenizer.lang_code_to_id[data_args.tgt_lang]
+        if isinstance(tokenizer, MBartTokenizer):
+            model.config.decoder_start_token_id = tokenizer.lang_code_to_id[data_args.tgt_lang]
+        else:
+            model.config.decoder_start_token_id = tokenizer.convert_tokens_to_ids(data_args.tgt_lang)
```

Comment:
@patrickvonplaten Wondering if this the right way to do this for the fast tokenizer, as it has no `lang_code_to_id`.

Reply:
this should work, as `MBartTokenizer` is inherited from `XLMRobertaTokenizer` and in `convert_tokens_to_ids` it first checks if the token is in `fairseq_tokens_to_ids` dict and returns the id if it is. All lang tokens are added in the `fairseq_tokens_to_ids` dict

Reply:
@patil-suraj, I think @sgugger was wondering about the fast tokenizers, so `MBartTokenizerFast`, and those don't have the `fairseq_tokens_to_ids` dict. However, I think your approach is the correct one here @sgugger . `MBartTokenizerFast` adds all languages codes, called `FAIRSEQ_LANGUAGE_CODES` to the special tokens so that they can be converted just the way you did above -> LGTM!

Reply:
Aah, I see. My bad.